downstream/titles/controller/controller-user-guide/master.adoc

```diff
@@ -0,0 +1,26 @@
+:imagesdir: images
+:numbered:
+:toclevels: 1
+
+:experimental:
+:controller-UG:
+
+include::attributes/attributes.adoc[]
+
+
+// Book Title
+= Automation Controller User Guide
+
+Thank you for your interest in {PlatformName} {ControllerName}. 
+{ControllerNameStart} helps teams manage complex multi-tier deployments by adding control, knowledge, and delegation to Ansible-powered environments.
+
+The {ControllerNameStart} User Guide describes all of the functionality available in {ControllerName}. 
+It assumes moderate familiarity with Ansible, including concepts such as Playbooks, Variables, and Tags. 
+For more information on these and other Ansible concepts, see the Ansible documentation at http://docs.ansible.com/. 
```

Comment:
The link redirects to https so perhaps the link itself should be changed to: https://docs.ansible.com/